### ISSUE TITLE
Rendering reprex fails with non-standard R prompt

### ISSUE THREAD

**Issue by jemus42** [tidyverse/reprex#166]
If you have a non-standard prompt set, the rendering of the reprex fails.

Example: ![](https://dump.jemu.name/2018-01-go4r1ent8e1mjxf.png)

Steps to reproduce:

- Make your prompt fun:

```
options(prompt = "\033[32m★\033[39m " ) # Makes prompt a green star
```

- Try to render any reprex 

Yes, I do realize this is kind of a silly issue and it's my own fault, and yes, I am aware I can (and indeed have) circumvent this issue by just resetting my prompt to the default prompt, but I just wanted to leave this here just in case anyone else stumbles upon this issue.

**Comment by jennybc** [comment]
Yeah, you're getting what you deserve 😼

It has already been brought to my attention by @gaborcsardi that I should be escaping more user input, though this is the first "organic" example of a nonstandard prompt that really causes problems.

**Comment by MilesMcBain** [comment]
`reprex` also fails when using `rice` which has `r$>` as default prompt.

**Comment by jennybc** [comment]
This has been fixed for a while via d1c20b13dbdaeb354ddd5b5463938bbd1314f5f5.
